requirements.txt

```diff
@@ -0,0 +1,45 @@
+aiohappyeyeballs==2.4.6
```

Comment:
requirements should not include all libraries that are dependencies of another library. for eg. click should not be here.